dcp_prototype/backend/wrangling/migrations/common/dataset_metadata.py

```diff
@@ -117,12 +129,21 @@ def process_donor_organism(self):
         for data in self.entity_data.get("donor_organism", []):
             self.process_mappings("donor_organism", data, self.project, mapping, self.missing)
 
+        # This is a special case for one project that lacks an ontology_label for genus_species.
+        # An alternative approach to solve this problem is to provide a list of source mappings for
+        # each destination mapping, in order of preference.  Then iterate through each source mapping
+        # until a match is found.  However, since this only impacts one attribute for one project,
+        # it's simpler to make a special case.
+        if not self.project.donor_species:
```

Comment:
Not really a comment on your code, but out of curiosity, is the `ontology_text` field populated for this project? I think I remember that there was a project where neither ontology nor ontology_label was populated, but ontology_text was.

Reply:
This one has "text" and "ontology" but no "ontology_text".